aldryn_jobs/forms.py

```diff
@@ -133,7 +133,7 @@ def is_edit_action(self):
 
     def validate_field_uniqueness_with_app_config(self, field_name,
                                                   error_message):
-        field = self.cleaned_data[field_name]
+        field = self.cleaned_data.get(field_name, '')
```

Comment:
it doesn't matter, since all we need to be sure that it wont produce 500 error, and that other validation errors would be shown. and if there is a value then filtering would be performed correctly, and other logic would make sense at that point. anyway filtering with empty string will not give any results because all slugs/names are namespaced and have a required fields (name and slug) and if slug is considered empty this form should populate it.